Code/Framework/AzNetworking/AzNetworking/aznetworking_headers.cmake

```diff
@@ -0,0 +1,77 @@
+#
```

Comment:
This turned into a pretty nasty change all together, gotta wonder if it's really worth it :\

Reply:
The alternative being, we just rip target manager out into a separate gem which on initial examination looks to be a pretty easy thing to do (very little to no coupling with the rest of azframework). If we did that, then we wouldn't have to do any of this to aznetworking.

Reply:
Ripping TargetManager out into a Gem is exactly what I'd like to do. It'll make quite a bit of the nastiness go away.